clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/agent/EddaLoadBalancerCachingAgent.groovy

```diff
@@ -88,6 +90,14 @@ class EddaLoadBalancerCachingAgent implements CachingAgent, HealthProvidingCachi
       String healthId = Keys.getInstanceHealthKey(ilb.instanceId, account.name, region, healthId)
       Map<String, Object> attributes = objectMapper.convertValue(ilb, ATTRIBUTES)
       Map<String, Collection<String>> relationships = [(INSTANCES.ns): [instanceId]]
+
+      CacheData instance = providerCache.get(INSTANCES.ns, instanceId, RelationshipCacheFilter.none())
+      if (instance != null) {
+        String application = instance.attributes.get("application")
+        if (application != null)
+          attributes.put("application", application)
```

Comment:
Stylistic preference here, but I prefer all conditionals to have squigglies, even if they're one-liners.